golang/cosmos/x/vtransfer/types/codec.go

```diff
@@ -0,0 +1,34 @@
+package types
+
+import (
+	"github.com/cosmos/cosmos-sdk/codec"
+	cdctypes "github.com/cosmos/cosmos-sdk/codec/types"
+	cryptocodec "github.com/cosmos/cosmos-sdk/crypto/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+var (
+	amino = codec.NewLegacyAmino()
+
+	// ModuleCdc references this module's global codec. Note, the codec should
+	// ONLY be used in certain instances of tests and for JSON encoding as Amino is
+	// still used for that purpose.
+	//
+	// The actual codec used for serialization should be provided to the module's
+	// keeper and defined at the application level.
+	ModuleCdc = codec.NewProtoCodec(cdctypes.NewInterfaceRegistry())
```

Comment:
`ModuleCdc` is unused AFAICT. But if it is needed for something, please reword the doc comment... I don't know how to interpret "_the codec should ONLY be used in certain instances of tests and for JSON encoding as Amino is still used for that purpose_" (specifically, I can't determine what "that purpose" refers to).
```suggestion
```

Reply:
I removed this file as it was blindly copied from an ancient Cosmos SDK example module.